src/section-square-root-properties.ptx

```diff
@@ -370,7 +370,7 @@
         <p>We learned the <xref ref="item-multiplication-property-of-square-roots" text="title"/> previously and applied this to multiplication of square roots, but we cannot apply this property to the operations of addition or subtraction. Here are two examples to demonstrate this:<md>
             <mrow>\sqrt{9+16}\amp\stackrel{?}{=}\sqrt{9}+\sqrt{16}\amp\sqrt{169-25}\amp\stackrel{?}{=}\sqrt{169}-\sqrt{25}</mrow>
             <mrow>\sqrt{25}\amp\stackrel{?}{=}3+4\amp\sqrt{144}\amp\stackrel{?}{=}13-5</mrow>
-            <mrow>5\amp\stackrel{no}{=}7\amp12\amp\stackrel{no}{=}8</mrow>
```

Comment:
Using \stackrel{no}{=} was an intentional choice we made last year, distinguished from \ne. It comes in situations like this, where previous lines had a \stackrel{?}{=}. It might be better if the "no" was some sort of "X", as a counter part to a \checkmark. We tried that at first, but didn't find a good "X" character.